src/main/kotlin/frc/robot/subsystems/climb/ClimbIOTalonFX.kt

```diff
@@ -36,21 +37,22 @@ class ClimbIOTalonFX : ClimbIO {
     }
 
     override fun openStopper() {
-        stopperMotor.set(TalonSRXControlMode.PercentOutput, ClimbConstants.STOPPER_MOTOR_POWER)
+        stopperMotor.set(TalonSRXControlMode.PercentOutput, -ClimbConstants.STOPPER_MOTOR_POWER)
     }
 
     override fun closeStopper() {
-        stopperMotor.set(TalonSRXControlMode.PercentOutput, -ClimbConstants.STOPPER_MOTOR_POWER)
+        stopperMotor.set(TalonSRXControlMode.PercentOutput, ClimbConstants.STOPPER_MOTOR_POWER)
     }
 
     override fun disableStopper() {
         stopperMotor.neutralOutput()
     }
 
     override fun updateInputs() {
-        inputs.stopperAppliedVoltage.mut_replace(stopperMotor.motorOutputVoltage, Units.Volt)
-        inputs.stopperCurrent.mut_replace(stopperMotor.statorCurrent, Units.Amps)
-        inputs.mainMotorAppliedVoltage.mut_replace(mainMotor.motorVoltage.value, Units.Volt)
-        inputs.isStopperStuck = stopperMotor.statorCurrent >= ClimbConstants.STOPPER_MOTOR_CURRENT_THRESHOLD
+        inputs.stopperAppliedVoltage = stopperMotor.motorOutputVoltage
+        inputs.stopperCurrent = stopperMotor.statorCurrent
+        inputs.mainMotorAppliedVoltage = mainMotor.motorVoltage.value
+        inputs.isStopperStuck = stopperMotor.statorCurrent.absoluteValue >= ClimbConstants.STOPPER_MOTOR_CURRENT_THRESHOLD.absoluteValue
```

Comment:
This is not an input but an output. Log the current as an input, compare it to the threshold in the subsystem.

Reply:
1cb59619e582a94a21d945886208dfb02c229f05